openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/flagencoders/WheelchairFlagEncoder.java

```diff
@@ -288,61 +414,52 @@ public int defineNodeBits(int index, int shift) {
+                return EncodingManager.Access.CAN_SKIP;
+            }
 
             // explicit motorroads are not usable
-            if (way.hasTag("motorroad", "yes"))
+            if (way.hasTag("motorroad", "yes")) {
+                if (debugSkippedWays) LOGGER.warn("way skipped (motorroad): " + way.getId() + " Tags: " + way.getTags().toString());
                 return EncodingManager.Access.CAN_SKIP;
+            }
 
             // do not get our feet wet, "yes" is already included above
-            if (isBlockFords() && (way.hasTag(KEY_HIGHWAY, "ford") || way.hasTag("ford")))
+            if (isBlockFords() && (way.hasTag(KEY_HIGHWAY, "ford") || way.hasTag("ford"))) {
+                if (debugSkippedWays) LOGGER.warn("way skipped (ford): " + way.getId() + " Tags: " + way.getTags().toString());
                 return EncodingManager.Access.CAN_SKIP;
-            
-            boolean bicycleOrHorseOnlyWay = (way.hasTag(KEY_BICYCLE, KEY_DESIGNATED) || way.hasTag(KEY_BICYCLE, KEY_OFFICIAL) || way.hasTag("horse", KEY_DESIGNATED) || way.hasTag("horse", KEY_OFFICIAL)) && !way.hasTag(restrictions, intendedValues);
-            if (bicycleOrHorseOnlyWay)
-                return EncodingManager.Access.CAN_SKIP;
-            
-            if (restrictedWheelchairHighways.contains(highwayValue)) {
-            	// In some countries bridleways cannot be travelled by anything other than a horse, so we should check if they have been explicitly allowed for foot or pedestrian
-                if (highwayValue.equals(KEY_BRIDLEWAY) && !(intendedValues.contains(way.getTag("foot", "no")) || intendedValues.contains(way.getTag(KEY_WHEELCHAIR, "no")))) {
-                    return EncodingManager.Access.CAN_SKIP;
-                }
-           		return EncodingManager.Access.WAY;
             }
-            
+
+
             if (fullyWheelchairAccessibleHighways.contains(highwayValue) || assumedWheelchairAccessibleHighways.contains(highwayValue) || limitedWheelchairAccessibleHighways.contains(highwayValue)) {
             	// check whether information on wheelchair accessbility is available
             	return EncodingManager.Access.WAY;
             }
-            
+
             // anything else
-            return EncodingManager.Access.CAN_SKIP;
+            return EncodingManager.Access.WAY;
         }
     }
 
     @Override
     public long handleRelationTags(long oldRelationFlags, ReaderRelation relation) {
         int code = 0;
-        if (relation.hasTag(KEY_ROUTE, "hiking") || relation.hasTag(KEY_ROUTE, "foot")) {
-            Integer val = hikingNetworkToCode.get(relation.getTag("network"));
-            if (val != null)
-                code = val;
+        if (relation.hasTag(KEY_ROUTE, "hiking")
+                || relation.hasTag(KEY_ROUTE, "foot")
+                || relation.hasTag(KEY_ROUTE, "bicycle")
```

Comment:
Should we "prefer" bike and inline skate tracks? I assume that in terms of physical attributes they are good for wheelchair, but I am not sure if there would be problems in terms of faster moving bikers and skaters coming across a wheelchair user in the path. I guess it all comes down to if the relation means that it is meant only for bikes/skaters or is just suitable for them